doc/source/data/transforming-datasets.rst

```diff
@@ -113,8 +113,8 @@ There are three types of UDFs that you can use with Ray Data: Function UDFs, Cal
   When using the actor compute strategy, per-row and per-batch UDFs can also be
   *callable classes*, i.e. classes that implement the ``__call__`` magic method. The
   constructor of the class can be used for stateful setup, and will be only invoked once
-  per worker actor. 
-  
+  per worker actor.
+
   Callable classes are useful if there is expensive state (such as a neural network) that need to be loaded for the UDF. By using an actor class, the state only needs to be loaded once in the beginning, rather than for each batch.
```

Comment:
```suggestion
  Callable classes are useful if you need to load an expensive state (such as a neural network) for the UDF. By using an actor class, the you only need to load the state once in the beginning, rather than for each batch.
```